src/actions.py

```diff
@@ -657,6 +658,16 @@ def on_message(bot: Bot, update: Update, user_data: dict, job_queue: JobQueue):
             with session_scope() as sess:
                 chat = Chat(id=chat_id, kick_timeout=timeout)
                 sess.merge(chat)
+            for job in job_queue.jobs():
+                context = job.context
+                timeout_value = context.get("timeout")
+                if timeout_value is not None:
```

Comment:
to understand which jobs you need to reschedule, you can use the `Job.name` param (see [docs](https://docs.python-telegram-bot.org/en/v12.2.0/telegram.ext.job.html))